readme.md

```diff
@@ -52,6 +52,12 @@ Type: `Object`
 
 Options are passed to the [`retry`](https://github.com/tim-kos/node-retry#retryoperationoptions) module.
 
+##### onFailedAttempt(err, attemptNo, attemptsLeft)
+
+Type: `Function`
+
+Callback invoked on each retry. Receives the error thrown by `input` as the first argument, the current attempt number as the second argument and the number of attempts left as the third argument.
```

Comment:
This needs to be updated to reflect the code (That it's no properties on the error).

I'd also like to see the docs here describe the use-case for this option, as commented in https://github.com/sindresorhus/p-retry/issues/3#issuecomment-344953574.